drivers/scheduler/k8s/specs/sysbench-vdbench/px-sysbench-vdbench-app.yml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: sysbench
+  labels:
+    app: sysbench
+spec:
+  selector:
+    matchLabels:
+      app: sysbench
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: sysbench
+    spec:
+      schedulerName: stork
+      containers:
+        - image: mysql:5.7
+          name: mysql
+          env:
+            - name: MYSQL_ROOT_PASSWORD
+              value: password
+          livenessProbe:
+            exec:
+              command: ["sh", "-c", "mysqladmin -u root -p$MYSQL_ROOT_PASSWORD ping"]
+            initialDelaySeconds: 70
+            periodSeconds: 10
+            timeoutSeconds: 5
+          readinessProbe:
+            exec:
+              command: ["sh", "-c", "mysql -u root -p$MYSQL_ROOT_PASSWORD -e \"select 1\""]
+            initialDelaySeconds: 60
+            periodSeconds: 10
+            timeoutSeconds: 5
+          volumeMounts:
+            - name: mysql-persistent-storage
+              mountPath: /var/lib/mysql
+        - name: sysbench
+          image: thiagogonzaga/sysbench:latest
```

Comment:
create a PTX to change this to portworx/torpedo-sysbench

Reply:
https://portworx.atlassian.net/browse/PTX-1535

Reply:
https://portworx.atlassian.net/browse/PTX-1536